src/persistance/fs-utils/index.ts

```diff
@@ -35,8 +35,31 @@ export const readDirIfExists = async (dirName: string): Promise<string[]> => {
   return fs.readdir(dirName);
 };
 
+export const readFileIfExists = async (location: string): Promise<unknown> => {
+  try {
+    const content = await fs.readFile(location, "utf-8");
+    return Promise.resolve(content);
+  } catch (e) {
+    return Promise.reject({});
+  }
+};
+
+export const createFile = async (
```

Comment:
this would overwrite the file if it already exists.
extract a common function `fileExists(filePath: string): Promise<boolean>` to make decisions in the two functions:

- readFileIfExists
- createFileIfNotExists (renamed one from createFile)

Follow the example of the dir related logic that I did.

Reply:
I am intentionally overwriting the file. I am reading the file before and then re-creating the entire file.